exchange/gdpr.go

```diff
@@ -0,0 +1,95 @@
+package exchange
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+// ExtractGDPR will pull the gdpr flag from an openrtb request
+func extractGDPR(bidRequest *openrtb.BidRequest, usersyncIfAmbiguous bool) (gdpr int) {
+	var re regsExt
+	var err error
```

Comment:
This `err` isn't used anywhere anymore.

Did you mean to `panic` on error in here?

Reply:
It is used ... `if ... || err != nil`